app/controllers/api/v2/config_reports_controller.rb

```diff
@@ -37,10 +37,8 @@ def show
       param_group :config_report, :as => :create
 
       def create
-        @config_report = ConfigReport.import(params[:config_report], detected_proxy.try(:id))
-        process_response @config_report.errors.empty?
-      rescue ::Foreman::Exception => e
-        render_message(e.to_s, :status => :unprocessable_entity)
+        job = ImportConfigReport.perform_later(params[:config_report], detected_proxy.try(:id))
```

Comment:
> is there any way to bypass rails from parsing the json?

I have done it in https://github.com/Katello/katello/pull/6640. In rails 4, we will need to reuse the [param_parser_wrapper](https://github.com/Katello/katello/blob/master/lib/katello/params_parser_wrapper.rb) from Katello and register it properly in the middleware.